pkg/apis/agent/v1alpha1/agent_types.go

```diff
@@ -24,6 +24,10 @@ const (
 	FleetServerServiceAccount commonv1.ServiceAccountName = "fleet-server"
 )
 
+var (
+	MandatoryPolicyIDVersion = version.MustParse("9.0.0-SNAPSHOT")
```

Comment:
Do we know it is going away in 9.0.0 or is this only a placeholder? If so let's add a comment.

Reply:
Hi @juliaElastic 👋 , could you confirm that `is_default` and `is_default_fleet_server` flags should be removed in `9.0.0`?

Reply:
Yes, that is the plan.